js/main.js

```diff
@@ -1,3 +1,4 @@
+new ghostCursor();
```

Comment:
When I tested this, it says ghostCursor() not defined. It looks like you are trying to create it without importing the other javascript file.